src/durablehttpresponse.ts

```diff
@@ -22,4 +22,18 @@ export class DurableHttpResponse {
             [key: string]: string;
         }
     ) {}
+
+    // returns the specified header, case insensitively
+    // returns undefined if the header is not set
+    public getHeader(name: string): string | undefined {
+        if (this.headers) {
+            const foundKey = Object.keys(this.headers).find(
+                (key) => key.toLowerCase() === name.toLowerCase()

```

Comment:
tiny optimization: move `name.toLowerCase()` outside the `.find()` operator :)

Reply:
Done!